### ISSUE TITLE
pdf-view

### ISSUE THREAD

**Issue by larrasket** [joaotavora/breadcrumb#34]
When opening a pdf file (w/ pdf-tools), breadcrumb shows a blank sequence. [screenshot](https://github.com/joaotavora/breadcrumb/assets/74098495/fcd6ac02-85bf-4cd8-9872-c4b7e5b28c17.

**Comment by joaotavora** [comment]
No idea what this means, or what you wanted to see.  Does `M-x imenu` work in any reasonable fashion in pdf-tools?

**Comment by larrasket** [comment]
I was expecting seeing the file name, similarly to how it does with
image-mode or fundamental-mode.  And yes imenue works fine with pdf-tool
when the pdf file is indexed.

**Comment by joaotavora** [comment]
You'll have to teach me how to get pdf-tool or pdf-view (which is it?) as I've never used anything of the sort.  Bonus points for an Emacs -Q recipe demonstrating the problem.

**Comment by joaotavora** [comment]
I'm using linux (Arch) so it should theoretically be simple to install, right?

**Comment by larrasket** [comment]
Yeah it is simple, doing yay -S emacs-pdf-tools-git should be it or check the installation guide here https://github.com/vedang/pdf-tools then M-x pdf-tools-install. I will try now reproducing the issue with Emacs -Q and see if it persists

**Comment by larrasket** [comment]
Couldn't reproduce.
